app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ReviewsController < ApplicationController
+  before_action :require_user, only: [:create]
+
+  def create
+    @review = Review.new(strong_params)
+    @video = Video.find(params[:video_id])
+    if @review.save
+      flash[:success] = 'you leave a review in this movie'
+      redirect_to video_path(@video)
+    else
+      flash[:warning] = 'You miss the body'
+      redirect_to video_path(@video)
+    end
+  end
+
+  def strong_params
+    (params.require(:review).permit(:rating, :body, :video_id, :user_id))
```

Comment:
The other braces are redundant. Also why are you supporting `:video_id, :user_id` in strong params? The purpose is to   let user have full control over those fields. But actually these ids should be set by your application, so you should not permit them.